collection/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "collection"
```

Comment:
I was wondering if we should call it `collection_with_id`. I know there is also the `Collection` structure but from what I understood, it's mostly an artifact used for `CollectionWithId` (even if we ended using both `Collection` and `CollectionWithId`).

Reply:
It doesn't matter just for now. The important is the name of the next repository. 
`collection` is already used in crates.io.
I would prefer something related to relations or graph. but let discuss that later